<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnOn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOff">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>71</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>631</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>631</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>631</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRx">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>1231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>271</width>
      <height>272</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buttonDebug">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Очистка лога</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Info</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonWarning">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Warning</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCritical">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Critical</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnTx">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TX</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFlag">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>351</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTx_2">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>490</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>60</pointsize>
     </font>
    </property>
    <property name="text">
     <string>copy</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>140</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
